test-butler-library/src/main/java/com/linkedin/android/testbutler/TestButler.java

```diff
@@ -232,6 +232,33 @@ public static void setRotation(@Rotation int rotation) {
         }
     }
 
+    /**
+     * A helper method for granting runtime permissions to apps under test on API 23+
+     * <p>
+     * Note: Before API 23, this method is a no-op
+     * <p>
+     * A common use case for this is when running tests from Android Studio where you can't (currently) pass the
+     * -g flag to adb install when installing the app to run tests.
+     *
+     * @param context the "target context"; e.g. Context of the app under test (not the test apk context!)
```

Comment:
I _think_ this should be i.e., not e.g.
Also, "app being tested" again. 
And maybe "not the instrumentation apk context!"